index.html

```diff
@@ -334,16 +365,23 @@
               <dd>error type is unknown</dd>
             </dl>
 
-            <p>The user agent MAY extend the above error type list with custom values - e.g. new error types to accommodate new protocols, or more detailed error descriptions of existing ones. When doing so, the user agent SHOULD follow the dot-delimited pattern (`[group].[optional-subgroup].[error-name]`) to facilitate simple and consistent processing of the error reports - e.g. the collector may provide aggregation by category and/or one or multiple subgroups.</p>
+            <p>The user agent MAY extend the above error type list with custom values — e.g. new error types to accommodate new protocols, or more detailed error descriptions of existing ones. When doing so, the user agent SHOULD follow the dot-delimited pattern (<code>[group].[optional-subgroup].[error-name]</code>) to facilitate simple and consistent processing of the error reports — e.g. the collector may provide aggregation by category and/or one or multiple subgroups.</p>
             </dd>
           </dl>
         </li>
 
-        <li>Return <em>neterror</em>.</li>
+        <li>Queue <em>neterror</em> for delivery to the <a data-lt="report-to">endpoint group</a> defined by the <a>NEL policy</a> of the associated <a>NEL origin</a>. Actual delivery of reports, including scheduling, retrying, or abandoning delivery, is handled by the Reporting API, not by Network Error Logging.</p>

```

Comment:
- Let's rename `neterror` to a different name, since we're proposing OK status too?
- "Queue x for delivery" should link to the algorithm in Reporting API. 
  - We can drop the "actual delivery..." once we have the link in place.

As an aside, you should be able to use `[[!REPORTING#id]]` for references.

Reply:
Done